pkg/config/components/honeycombExporter.yaml

```diff
@@ -23,6 +23,14 @@ properties:
     type: string
     validations: [nonblank, url]
     default: https://api.honeycomb.io
+  - name: Compression
+    summary: The compression to use when sending data to Honeycomb
+    description: |
+      The compression to use when sending data to Honeycomb.
+      This is normally gzip, but can be overridden.
+    type: string
+    validations: [nonblank]
```

Comment:
Does having this as `nonblank` mean you have to provide a value. Is `none` okay?